src/baldr/incident_singleton.h

```diff
@@ -0,0 +1,306 @@
+#pragma once
+
+#include "baldr/graphreader.h"
+#include "midgard/sequence.h"
+
+#include <chrono>
+#include <condition_variable>
+#include <ctime>
+#include <memory>
+#include <mutex>
+#include <thread>
+
+#include <google/protobuf/io/coded_stream.h>
+#include <google/protobuf/io/zero_copy_stream_impl_lite.h>
+
+namespace {
+
+#ifdef _MSC_VER
+#define MTIME(st_stat) st_stat.st_mtime
+#elif __APPLE__
+#define MTIME(st_stat) st_stat.st_mtime
+#else
+#define MTIME(st_stat) st_stat.st_mtim.tv_sec
+#endif
+
+constexpr time_t DEFAULT_MAX_LOADING_LATENCY = 60;
```

Comment:
Nit: setting the `time_t` value may not be portable

> Portable programs should not use values of this type directly, but always rely on calls to elements of the standard library to translate them to portable types.

Reply:
its not portable in the sense that we dont know how many bits wide that it is. for this use we dont care, we just care that its the arithmetic type that comes out of the `time` function